packages/one-app-bundler/bin/bundle-module.js

```diff
@@ -13,22 +13,33 @@
  * under the License.
  */
 
-if (process.env.NODE_ENV !== 'production' && process.argv.includes('--dev')) {
-  console.info('Running dev bundler');
-  import('@americanexpress/one-app-dev-bundler').then(({ default: esbuildBundleModule }) => {
-    if (esbuildBundleModule) {
-      esbuildBundleModule().catch((error) => {
+const bundleModule = async () => {
+  const {
+    devBuildModule,
+    bundleExternalFallbacks,
+  } = await import('@americanexpress/one-app-dev-bundler');
+
+  console.info('Bundling External Fallbacks');
```

Comment:
Can we log this inside of `bundleExternalFallbacks` AFTER we verify that there are externals?